packages/shared-utils/lib/i18n/config.ts

```diff
@@ -177,12 +235,17 @@ const copyCodeConfig: PluginI18nConfig<CopyCodeI18NConfig> = {
     copy: "Copy successfully 🎉",
     hint: "Copy the code",
   },
+  "/en/": {
```

Comment:
Could you changed it to `/vi/`? It seems to be a typo? Thanks!

Reply:
Sorry, my typo